.github/workflows/auth-e2e-client-tests.yml

```diff
@@ -23,24 +23,22 @@ jobs:
           touch ${ENV_FILE_PATH}
 
           # Secrets (can't load these from another job, due to GH security features)
-          echo AUTH0_CLIENT_SECRET=${{ secrets.AUTH0_CLIENT_SECRET }} >> ${ENV_FILE_PATH}          
+          echo AUTH0_CLIENT_SECRET=${{ secrets.AUTH0_CLIENT_SECRET }} >> ${ENV_FILE_PATH}
 
       - name: Cypress run (Standard suite)
         uses: cypress-io/github-action@v6
         env:
-          CYPRESS_AUTH_PASSWORD_WITH_KEY_ROLES: ${{ secrets.AUTH_PASSWORD_WITH_KEY_ROLES }}
-          CYPRESS_AUTH_USERNAME_WITH_KEY_ROLES: ${{ vars.AUTH_USERNAME_WITH_KEY_ROLES }}
-          CYPRESS_AUTH_PASSWORD_WITHOUT_KEY_ROLES: ${{ secrets.AUTH_PASSWORD_WITHOUT_KEY_ROLES }}
-          CYPRESS_AUTH_USERNAME_WITHOUT_KEY_ROLES: ${{ vars.AUTH_USERNAME_WITHOUT_KEY_ROLES }}
```

Comment:
The authentication-service tests started failing and after looking into it I think it's because the variables were using Tyson's users which look like they've been deleted from Auth0. After figuring out how to fix it I realized that some variables were not used so I deleted them and the other ones had a vague name "key roles" so I switched it to something more clear "widget role"